build_tools/python/benchmark_suites/iree/x86_64_benchmarks.py

```diff
@@ -40,44 +40,33 @@ def _generate_default(
       self, device_specs: List[common_definitions.DeviceSpec]
   ) -> Tuple[List[iree_definitions.ModuleGenerationConfig],
              List[iree_definitions.E2EModelRunConfig]]:
-    # Create benchmarks for small workloads.
-    gen_configs_small = [
-        iree_definitions.ModuleGenerationConfig.build(
-            compile_config=self.CASCADELAKE_COMPILE_CONFIG,
-            imported_model=iree_definitions.ImportedModel.from_model(model))
-        for model in model_groups.SMALL
-    ]
-    execution_configs_small = [
-        module_execution_configs.get_elf_local_task_config(1),
-        module_execution_configs.get_elf_local_task_config(8)
-    ]
-    run_configs_small = benchmark_suites.iree.utils.generate_e2e_model_run_configs(
-        module_generation_configs=gen_configs_small,
-        module_execution_configs=execution_configs_small,
-        device_specs=device_specs)
-
-    # Create benchmarks for large workloads.
-    gen_configs_large = [
-        iree_definitions.ModuleGenerationConfig.build(
-            compile_config=self.CASCADELAKE_COMPILE_CONFIG,
-            imported_model=iree_definitions.ImportedModel.from_model(model))
-        for model in model_groups.LARGE
-    ]
-    # We use higher thread counts for large workloads.
-    execution_configs_large = [
-        module_execution_configs.get_elf_local_task_config(1),
-        module_execution_configs.get_elf_local_task_config(8),
-        # We use (MAX_CORE - 2) here to determine the maximum number of threads to use on a NUMA node.
-        # 13 threads are currently disabled until we switch to using c2-standard-60.
-        # module_execution_configs.get_elf_local_task_config(13),
-    ]
-    run_configs_large = benchmark_suites.iree.utils.generate_e2e_model_run_configs(
-        module_generation_configs=gen_configs_large,
-        module_execution_configs=execution_configs_large,
-        device_specs=device_specs)
-
-    return (gen_configs_small + gen_configs_large,
-            run_configs_small + run_configs_large)
+    gen_configs_all = []
+    run_configs_all = []
+
+    for model_config in model_groups.x86_64_MODELS:
+      gen_config = iree_definitions.ModuleGenerationConfig.build(
+          compile_config=self.CASCADELAKE_COMPILE_CONFIG,
+          imported_model=iree_definitions.ImportedModel.from_model(
+              model_config[0]))
+
+      execution_configs = []
+      for thread in model_config[1]:
+        if thread == 0:
+          execution_configs.append(
+              module_execution_configs.ELF_LOCAL_SYNC_CONFIG)
```

Comment:
This variable is undefined, I think? Do you mean `execution_configs`?

Reply:
Hmm.. Code already shows `execution_configs`?

Reply:
Weird, I was seeing `module_execution_configs`. Maybe I was somehow not viewing all commits?

![image](https://user-images.githubusercontent.com/5732088/230195269-f9b45948-c35e-46f1-a953-2ec35d143297.png)